learning_journal/templates/layout.jinja2

```diff
@@ -19,6 +21,13 @@
                     <a href="/">Home</a>
                     <br>
                     Learning Blog
+                    <h3>
+                        {% if request.authenticated_userid == "jordan" %}
```

Comment:
Avoid using hard coded usernames (or passwords) anywhere in your app.  This makes an app extremely rigid and a pain to adapt for new deployments. (or to scale to allow multiple users).  There is a better way to write this statement.